Project/Back-End/applicant_app/models.py

```diff
@@ -4,7 +4,8 @@
 from user_app.models import User
 # Create your models here.
 class Applicant(models.Model):
-    email = models.OneToOneField(User, on_delete=models.CASCADE)
+    email = models.OneToOneField(User.email, on_delete=models.CASCADE)
+    name = models.OneToOneField(User.name, on_delete=models.CASCADE)
```

Comment:
Is it possible to have a OneToOneField to a user and then you could just grab the information needed in your serializer? Just a suggestion if this works and it fits your needs best then please ignore